README.md

```diff
@@ -8,7 +8,7 @@ Qiskit is made up of elements that each work together to enable quantum computin
 
 ## Installation
 
-We encourage installing Qiskit via the PIP tool (a python package manager), which installs all Qiskit elements, including this one.
+We encourage installing Qiskit via the PIP tool (a python package manager). The following command installs the core Qiskit components, including Aer.
```

Comment:
Can you change `PIP` to lowercase `pip` just for consistency with Terra: https://github.com/Qiskit/qiskit-terra/pull/7476/files

Reply:
Changed in 0a80a422fed08e449116b5d73a3e0694b1b0ec39.

Note that [the line immediately after the command](https://github.com/Qiskit/qiskit-aer/blob/f778f4b53b4223c86f5e41b50794d176208c76c6/README.md?plain=1#L17) has the spelling PIP too.  I believe this should be "Pip" since it begins a sentence.  Shall I update it too?  On the other hand, I did not touch it when I first made this PR, as this same sentence appears in the READMEs for terra, ibmq-provider, ignis, and metal as well.

Reply:
May as well update it now, while we're at it, thanks - there's less review and CI overhead if we fix several typos in one go, rather than spreading them over loads of PRs (and "fix typos" is a valid single topic for a PR).

Reply:
Great -- fixed accordingly.